### PR TITLE
fix(feed-item-row, emoji-row): spacing / color fixes

### PR DESCRIPTION
**Pull request by braddialpad**
# fix(feed-item-row, emoji-row): spacing / color fixes

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1185

## :book: Description

Some final changes for feed item row / emoji row requested by design. Hopefully will be the last ones before this goes into master.

## :bulb: Context

Improvements for feed item row in product.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.

For all CSS changes:

- [x] I have used design tokens whenever possible.
- [x] I have considered how this change will behave on different screen sizes.
- [x] I have visually validated my change in light and dark mode.
- [x] I have used gap or flexbox properties for layout instead of margin whenever possible.

## :crystal_ball: Next Steps

Update in product branch.

### CODE REVIEW

File: packages/dialtone-vue2/.storybook/preview-head.html [hunk 1]
```diff
@@ -2,4 +2,8 @@
   .sbdocs.sbdocs-content {
     max-width: 2000px;
   }
+
+  * {
+    box-sizing: border-box;
+  }
```
Comment (braddialpad) [inline packages/dialtone-vue2/.storybook/preview-head.html:8]: Noticed we didn't have box-sizing: border-box set globally in storybook, causing inconsistencies with spacing between storybook and product.

File: packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue [hunk 2]
```diff
@@ -122,16 +122,20 @@ export default {
...
       color: var(--dt-color-link-primary);
       border-width: var(--dt-size-border-150);
-      background-color: var(--dt-color-purple-100) !important;
-      border-color: var(--dt-color-brand-purple) !important;
+      font-weight: var(--dt-font-weight-bold);
+      background-color: var(--dt-color-purple-100);
+      border-color: var(--dt-color-brand-purple);
+      .dt-emoji-row__reaction-number {
+        font-weight: var(--dt-font-weight-bold);
+      }
```
Comment (braddialpad) [inline packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue:135]: had to increase specificity here or `var(--dt-typography-body-small);` would override this.

**Comment by braddialpad** [comment]
will put up vue 3 after approval

File: packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue [hunk 3]
```diff
@@ -142,9 +145,7 @@ export default {
   }
 
   &__reaction-number {
-    font-weight: var(--dt-font-weight-bold);
-    font-size: var(--dt-font-size-100);
+    font: var(--dt-typography-body-small);
```
Comment (francisrupert) [inline packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue:148]: Add `line-height: var(--dt-font-line-height-100);` in the line after `font`.

This should get the button to exactly 24px tall.

The Emoji icon is a smaller size than the native emoji in the Figma file, the height of the button is out of whack w/ the spacing via padding and the "body-small" style.

**Review by francisrupert — CHANGES_REQUESTED** [review]
I suppose you should check with @bhawkins-dialpad too, though it might be a bit awkward outside of context from the entire feed. 

Made a small tweak to ensure the emoji row button is correct size.

File: packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue (hunk 3 above)
Comment (braddialpad) [reply packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue:148]: nice thanks.

**Review by francisrupert — APPROVED** [review]
LGTM, but I'll defer approval to others for the Vue stuffs. I don't want to block you, so let's get this going so you can update in your product branch and then informally throw it by Bradley.

**Pull request merged**